README.md

```diff
@@ -199,7 +198,7 @@ git checkout $(cat ../cts_runner/revision.txt)
 To run a given set of tests:
 
 ```
-# Must be inside the cts folder we just checked out, else this will fail
+# Must be inside the CTS folder we just checked out, else this will fail
```

Comment:
```suggestion
# Must be inside the `cts` folder we just checked out, else this will fail
```

This is a literal folder name, so must be of the same case